suite-native/transactions/src/components/TransactionsEmptyState.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Image } from 'react-native';
+
+import { useAtom } from 'jotai';
+
+import { Box, Button, Card, Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { isSendReceiveActionsVisibleAtom } from '@suite-native/navigation';
+
+const cardStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'center',
+    alignItems: 'center',
+    flex: 1,
+    width: '100%',
+    paddingHorizontal: utils.spacings.large,
+    borderRadius: utils.borders.radii.large,
+}));
+
+const receiveButtonStyle = prepareNativeStyle(() => ({
+    width: 310,
```

Comment:
Are we sure we want fixed width here? Would be some kind of percentage better?

Reply:
Changed to percentage. If there will be problem, we can always use fixed size.